ratchet/ratchet.go

```diff
@@ -43,9 +44,9 @@ const (
 // Ratchet contains the per-contact, crypto state.
 type Ratchet struct {
 	MyPrivateKey        *[sntrup4591761.PrivateKeySize]byte
-	MySigningPublic     *[32]byte
+	MySigningPublic     *ed25519.PublicKey
```

Comment:
`ed25519.PublicKey` is type `[]byte`, not an array. the extra pointers throughout are unnecessary